.golangci.yml

```diff
@@ -35,11 +28,7 @@ linters:
 
 linters-settings:
   errcheck:
-    exclude: .errcheck-excludes
-  gocritic:
-    enabled-checks:
-      # Detects suspicious append result assignments. E.g. "b := append(a, 1, 2, 3)"
-      - appendAssign
```

Comment:
Is it covered by default in the new version?

Reply:
Yes it is! I got a warning that the config is redundant:
```
WARN [linters_context] gocritic: no need to enable check "appendAssign": it's already enabled 
```